flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/runway.FLT

```diff
@@ -160,9 +160,9 @@ A32NX_ELEC_IDG1LOCK_TOGGLE=0
 A32NX_ELEC_IDG2LOCK_TOGGLE=0
 A32NX_KNOB_OVHD_AIRCOND_XBLEED_Position=1
 A32NX_KNOB_OVHD_AIRCOND_PACKFLOW_Position=1
-A32NX_AIRCOND_PACK1_FAULT=0
+A32NX_OVHD_COND_PACK_1_PB_HAS_FAULT=0
 A32NX_OVHD_COND_PACK_1_PB_IS_ON=1
-A32NX_AIRCOND_PACK2_FAULT=0
+A32NX_OVHD_COND_PACK_2_PB_HAS_FAULT=0
```

Comment:
**thought**: As these simvars are managed by the Rust simulation, there's probably no need to keep them in the `*.flt`  files.

Reply:
Again absolutely right, removed from all FLT files.